includes/Products.php

```diff
@@ -479,7 +480,7 @@ public static function is_commerce_enabled_for_product( \WC_Product $product )
 	 */
 	public static function update_commerce_enabled_for_product( \WC_Product $product, $is_enabled ) {
 
-		// TODO: implement
+		$product->update_meta_data( Products::COMMERCE_ENABLED_META_KEY, wc_bool_to_string( $is_enabled ) );
```

Comment:
How about we also `save_meta_data()` here like the other update methods, as right now it won't be persisted